### PR TITLE
add dummy list organisiations and tests

### PR DESCRIPTION
**Pull request by idrysdale**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -460,8 +460,47 @@ HTTP 204 No content
...
 
+## List my organisations
+
+```
+GET /organisations/
+```
+
+List all organisations the logged in user is part of
+
+ℹ️ porting notes: replaces previous `assessment/list` route, passing `orgid`.
```
Comment (paulfurley) [inline README.md]: `assessment/list` with `orgid` lists *assessments* within an organisation, not my organisations

Comment (paulfurley) [reply README.md]: `assessment/getorganisations`

Comment (idrysdale) [reply README.md]: Fixed! Thanks!

File: README.md [hunk 2]
```diff
@@ -460,8 +460,47 @@ HTTP 204 No content
...
+
+Returns:
+
+```
+HTTP 200 OK
+Content-Type: application/json
+
+[
+    {
+        "orgid": "1",
```
Comment (paulfurley) [inline README.md]: this should just be `id`

File: mhep/mhep/assessments/tests/views/test_dummy_organisation_endpoints.py [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+
+class TestListMyOrganisations(APITestCase):
+    def test_list_organisation_assessments(self):
+        response = self.client.get("/api/v1/organisations/")
+        assert response.status_code == status.HTTP_200_OK
+
+        expected = [
+            {
+                "orgid": "1",
```
Comment (paulfurley) [inline mhep/mhep/assessments/tests/views/test_dummy_organisation_endpoints.py]: `orgid` -> `id`

**Pull request merged**